tests/queries/0_stateless/03532_window_function_andnull_source_max_threads.reference

```diff
@@ -0,0 +1,3 @@
+  Resize 20 → 20
```

Comment:
Let's trim leading spaces, so the test will not be affected if some steps will be added/removed between `Resize`.